fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/RowSpliterator.java

```diff
@@ -105,6 +108,16 @@ private Row next() throws XMLStreamException {
     private Cell parseCell() throws XMLStreamException {
         String cellRef = r.getAttribute("r");
         CellAddress addr = new CellAddress(cellRef);
+
+        CellAddress mergedCellAddress = null;
+
+        for(CellRangeAddress mergedCell : mergedCells) {
+            if (mergedCell.isInRange(addr.getRow(), addr.getColumn())) {
+                mergedCellAddress = new CellAddress(mergedCell.getFirstRow(), mergedCell.getFirstColumn());
+                break;
+            }
+        }
```

Comment:
I am afraid this could introduce a huge performance regression if there are many merged cell ranges, Maybe we could organize them in an ordered data structure to speed up this search? For an example, see #236.